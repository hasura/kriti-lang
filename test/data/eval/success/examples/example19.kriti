# If Statement Examples
[
  {{ if ((1) != 3) }} true {{ else }} false {{ end }},
  #{{ if not ((1) != 3) }} true {{ else }} false {{ end }},
  {{ if ((1) >= 3) }} true {{ else }} false {{ end }},
  {{ if ((1) <= 3) }} true {{ else }} false {{ end }},
  {{ if "foo" in ["foo"] }} 1 {{ else }} 0 {{ end }},
  {{ if "foo" in { "foo": true } }} 1 {{ else }} 0 {{ end }},
  {{ if ("foo" in { "foo": true }) }} 1 {{ else }} 0 {{ end }}
]
