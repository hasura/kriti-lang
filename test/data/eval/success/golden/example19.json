[
    true,
    false,
    false,
    true,
    1,
    1,
    1
]